srml/contracts/src/wasm/mod.rs

```diff
@@ -1634,4 +1662,103 @@ mod tests {
 		assert_eq!(output, ExecReturnValue { status: 17, data: hex!("5566778899").to_vec() });
 		assert!(!output.is_success());
 	}
+
+	const CODE_GET_RUNTIME_STORAGE: &str = r#"
```

Comment:
Have you thought about using `include_bytes!` instead of embedding all those raw strings into the actual code?

Reply:
Yeah, it's one of the options. One another is to move those declarations in a separate `.rs` file. It is useful because some wasm test fixtures are shared between integration tests and unit tests. We can even combine these two approaches. 

However, I think this is better done as a follow up